src/pages/filter/ui/modal-edit/modal-edit.store.ts

```diff
@@ -47,22 +59,64 @@ class ModalEditStore {
     ].length
   }
 
+  public get currentStepGroups(): string[] {
+    return dtreeStore.currentStepGroups
+  }
+
   public get problemGroups(): string[] {
-    let attrData: any
+    let problemGroups: string[] = []
 
     const subGroups = Object.values(
       getQueryBuilder(dtreeStore.startDtreeStat['stat-list']),
     )
 
+    subGroups.forEach(subGroup => {
+      subGroup.find((item, currNo) => {
```

Comment:
The `find` method returns found element, try use it (e.g. `const group = subGroup.find(item => item.name === this.groupName)`)

Also, converting `stat-list` to dictionary is note required here. 
You can just iterate over `stat-list` to find the group.
`return dtreeStore.startDtreeStat['stat-list'].find(item => item.name === this.groupName)?.family ?? []`